x-pack/test/security_solution_cypress/cypress/urls/edit_rule.ts

```diff
@@ -5,6 +5,6 @@
  * 2.0.
  */
 
-export function editRuleUrl(ruleId: string): string {
+export function ruleEditUrl(ruleId: string): string {
```

Comment:
Is this change really necessary? Changing this modifies 2 files owned by the @elastic/security-threat-hunting-investigations team but I'm not really seeing any value...

Reply:
Undoing the change would lower the number of files modified in this PR

Reply:
I agree. Should we go as far as renaming the file then? From `edit_rule.ts` to `rule_edit.ts`?